src/koa.ts

```diff
@@ -1,17 +1,8 @@
 import { JSONSchema4 } from 'json-schema';
 import type { ParameterizedContext } from 'koa';
-import type Router = require('koa-router');
+import type Router from 'koa-router';
 import type { EndpointsOf, IntrospectionResponse, OneSchema } from './types';
 
-// This declare is required to override the "declare" that comes from
-// koa-bodyparser. Without this, the typings from one-schema will be
-// overriden and collapsed into "any".
-declare module 'koa' {
-  interface Request {
-    body?: unknown;
-  }
-}
-
```

Comment:
I simply can't reproduce the original error I described here. I've tried in every project currently using `one-schema`. I think it can go.